.ansible-lint

```diff
@@ -0,0 +1,6 @@
+---
+profile: production
+
+exclude_paths:
+  - changelogs/changelog.yaml
```

Comment:
I guess we should ansible-lint the changelog file. We do that in the other Validated collections if I am not mistaken.